Exercise_B/index.html

```diff
@@ -26,9 +7,76 @@
 
     <!-- Payments Div -->
     <!-- Consultation Div -->
-
     <!-- Delivery Div -->
-
     <!--Scripts required for bootstrap CDN-->
+
+
+
+<!DOCTYPE html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title></title>
+    <meta name="description" content=""/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/style/stylesheet.css">
+  </head>
+  <body>
+    <!-- CODE HERE -->
+    
+    <!-- Nav -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid gx-9">
+        <a class="navbar-brand" href="#">
+          <img src="./assets/images/logo.png" alt="">
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+          </div>
+        </div>
+      </div>
+    </nav>
+
+    <!-- Hero -->
+    <div class="d-flex align-items-center text-center hero">
+      <div class="container-fluid py-5">
+        <h1 class="display-5 fw-bold heading-words">Beatiful Flowers</h1>
+        <button class="btn btn-success btn-lg" type="button">Learn More</button>
+      </div>
+    </div>
+
+    <!-- Features -->
+    <div class="container-fluid features-container pt-5 pb-5">
+      <div class="container">

```

Comment:
Great use of the Bootstrap utilities here! With this in mind fantastic use of the row-cols-2 system.
This is a difficult system to integrate properly so great job.